client/src/components/goals-list/goals-list.js

```diff
@@ -3,25 +3,37 @@ import PropTypes from 'prop-types';
 import Goal from '../goal';
 import KeyListener from '../key-listener';
 
+const isShowOwnersEnabled = () => {
+  return localStorage.getItem('show_story_owners') === 'true';
```

Comment:
So this will be false by default? Is that what you meant?

Reply:
Yeah. I kind of want to keep them hidden by default for now and see if people like or or if it's too much clutter in the UI.

Reply:
makes sense 👍